JavaAssignment/src/kunal/SquareCube.java

```diff
@@ -0,0 +1,21 @@
+package kunal;
+public class SquareCube
+{
+	void Display(float number)
+	{
+		float Square;
+		float Cube;
+		Square = number * number;
```

Comment:
The way you declare the variable is not wrong, but not appropriate. The above thing simply adding multiple lines in code. 
Define directly like 
float square = number * number ;

Reply:
local variable should be initialized.